deploy/cr.yaml

```diff
@@ -87,6 +87,19 @@ spec:
 #      topologyKey: kubernetes.io/hostname
 #      whenUnsatisfiable: DoNotSchedule
 
+    exposePrimary:
+      enabled: true
+#      type: ClusterIP
```

Comment:
It looks like we need to update helm too.

Reply:
https://github.com/percona/percona-helm-charts/pull/575